packages/steveo/src/runtime/workflow.ts

```diff
@@ -0,0 +1,378 @@
+import { v4 } from 'uuid';
+import bind from 'lodash.bind';
+import assert from 'node:assert';
+import nullLogger from 'null-logger';
+import take from 'lodash.take';
+import { Step, StepUnknown } from './workflow-step';
+import { IProducer, IRegistry, Logger } from '../common';
+import { WorkflowState } from './workflow-state';
+import { Repositories, Storage } from '../storage/storage';
+import { WorkflowOptions, WorkflowPayload } from '../types/workflow';
+
+interface ExecuteContext {
+  workflowId: string;
+
+  payload: unknown;
+
+  step: StepUnknown;
+
+  state: WorkflowState;
+
+  repos: Repositories;
+}
+
+export class Workflow {
+  $name: string;
```

Comment:
what is the `$` approach?

Reply:
I saw this style of '$' leading private variable declarations in Payments and thought it was "at thing".... I'm not a fan myself either and on reviewing Google's guidelines they aren't either.... I'll remove them